run_dir/static/js/running_notes.js

```diff
@@ -143,34 +144,55 @@ function filter_running_notes(search){
         }
     });
 }
-//Filter notes by Category
+//Filter notes
 $('#rn_search').keyup(function() {
     var search=$('#rn_search').val();
     filter_running_notes(search);
 });
-$('.btnCatFilter').click(function() {
-    $('.btnCatFilter').each(function() {$(this).removeClass("glow")});
-    $(this).addClass("glow")
-    var search=$(this).text();
-    if (search == 'All'){
-        search='';
-    }
-    filter_running_notes(search);
+
+$(document).ready(function() {
+    $('#rn_category').change(function() {
+        var search=$('#rn_category :selected').text();
+        if (search == 'All'){
+            search='';
+        }
+        filter_running_notes(search);
+    });
 });
+
+// Update the category buttons
+$('.btnCatFilter').click(function(){
+    var was_selected = $(this).hasClass('active');
+    $('.btnCatFilter').removeClass('active');
+    if(!was_selected){ $(this).addClass('active'); }
+    });
```

Comment:
* Update with new class name
* Fix whitespace
* Add call to running-note preview function.

```suggestion
// Update the category buttons
$('.rn-cat-filter button').click(function(){
    var was_selected = $(this).hasClass('active');
    $('.rn-cat-filter button').removeClass('active');
    if(!was_selected){
        $(this).addClass('active');
    }
    preview_running_notes();
});
```

Reply:
The .click is already triggering the preview button here:
`$('.rn-cat-filter button').click(preview_running_notes());`